elements/pl-file-editor/pl-file-editor.js

```diff
@@ -179,21 +207,37 @@ window.PLFileEditor.prototype.initSettingsButton = function (uuid) {
   this.saveSettingsButton.click(function () {
     var theme = that.modal.find('#modal-' + uuid + '-themes').val();
     var fontsize = that.modal.find('#modal-' + uuid + '-fontsize').val();
-    that.editor.setTheme(theme);
-    that.editor.setFontSize(fontsize);
+    var keyBind = that.modal.find('#modal-' + uuid + '-keyboardHandler').val();
+
     localStorage.setItem('pl-file-editor-theme', theme);
     localStorage.setItem('pl-file-editor-fontsize', fontsize);
+    localStorage.setItem('pl-file-editor-keyboardHandler', keyBind);
```

Comment:
```suggestion
    localStorage.setItem('pl-file-editor-keyboardHandler', keyboardHandler);
```

Reply:
I will replace all the naming that contains "keyBinding" with "keyboardHandler"